set-contract-address.js

```diff
@@ -1,5 +1,35 @@
+/*
+ * Script to set addressresolver to deployed FlightDelayNewPolicy contract address
+ *
+ */
+
+
+const network = process.argv[process.argv.length - 1];
+
+const network_params = {
+  kovan: {
+    address: '0xa1eee690186f36e2b6d1d3ac17ee9ac7e503a0d3',
+//    address: '0x9e8e87e586e1337ac3207bd36f61a0e44619a24c',
+    networkId: 42,
+    rpcport: 8745,
```

Comment:
If you want to use different ports for each network it should be reflected in deploy scripts as well